tutor/specs/components/exercises/details.spec.jsx

```diff
@@ -17,6 +17,7 @@ describe('Exercise Details Component', function() {
       book,
       exercises,
       pageIds,
+      selectedSection:        book.chapter_section,
```

Comment:
I think I removed the `isRequired` for this in the ModalManager PR btw (because a spec kept complaining).

Reply:
ah, I probably hadn't pulled master.    is still good to add it.